WIP - implement a wider layout

cdccollins opened the pull request.

cdccollins committed:
Add Vicky Peele to people page

cdccollins committed:
Add Igor Fontana to people page

cdccollins committed:
Re-arrange people

cdccollins committed:
Add Ben Baumann to people page

neilvanbeinum committed:
Add this week's roundup

cdccollins committed:
Ben is no longer blurry

cdccollins committed:
WIP - implement a wider layout

cdccollins committed:
WIP - implement a wider layout

ubxd commented:
Deploy preview for *unboxed-co-production* ready!

Built with commit 9911e4a67702eaf6e594a64d1e7595ae09715b8d

https://deploy-preview-529--unboxed-co-production.netlify.com

cdccollins committed:
Change desktop-breakpoint variable and keep blog in smaller width

borisdivjak committed:
update homepage logos for wider layout

borisdivjak committed:
updated logos on ruby-on-rails services page

cdccollins committed:
Replaced images and amended team stylesheet

Teams of 3 became stretched in new layout, so deleted some styling that was affecting this

borisdivjak committed:
ruby-on-rails-development images updated

borisdivjak committed:
ruby-on-rails-development images update done

borisdivjak committed:
southwark ccg images update

borisdivjak committed:
service pages images updated

borisdivjak committed:
v-and-a images updated

borisdivjak committed:
reed-learning images update

borisdivjak committed:
sh24 images update

cdccollins committed:
Merge branch 'WIP-implement-wider-layout' of github.com:unboxed/unboxed.co into WIP-implement-wider-layout
